recipes/kwant/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "kwant" %}
+{% set py3k_ver = "1.2.2" %}
+{% set py2k_ver = "1.1.2" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name }}
+  version: {{ py3k_ver }}  # [py3k]
+  version: {{ py2k_ver }}  # [py2k]
```

Comment:
Development for Python 2 stopped, however, bug fixes will still be applied. So the version number for both Python 2 and 3 will change, but not be the same.